File: Projetos/ACBrLib/Demos/C#/NFe/Imports/Dinamico/Shared/NotaFiscal/IBSCBSTotNFe.cs
namespace ACBrLib.NFe
{
    public class IBSCBSTotNFe
    {
        #region Constructors
        public IBSCBSTotNFe()
        {
            gIBS = new gIBSTot();
            gCBS = new gCBSTot();
            gMono = new gIBSCBSMonoTot();

        }
        #endregion Constructors
        #region Properties

        /// <summary>
        /// Valor total da BC do IBS e da CBS
        /// </summary>
        public decimal vBCIBSCBS { get; set; }

        /// <summary>
        /// Grupo total do IBS
        /// </summary>
        public gIBSTot gIBS { get; }

        /// <summary>
        /// Grupo total do CBS
        /// </summary>
        public gCBSTot gCBS { get; }

        public gIBSCBSMonoTot gMono { get; }
       

        #endregion Properties
    }
}

File: Projetos/ACBrLib/Android/NFe/ACBrLibNFe/src/main/java/br/com/acbr/lib/nfe/notafiscal/IBSCBSTotNFe.java
package br.com.acbr.lib.nfe.notafiscal;

import br.com.acbr.lib.comum.dfe.gCBSTot;
import br.com.acbr.lib.comum.dfe.gIBSTot;
import br.com.acbr.lib.nfe.gIBSCBSMonoTot;

public class IBSCBSTotNFe {

    public IBSCBSTotNFe(){
        gIBS = new gIBSTot();
        gCBS = new gCBSTot();
        gMono = new gIBSCBSMonoTot();
    }

    private double vBCIBSCBS;
    private gIBSTot gIBS;
    private gCBSTot gCBS;
    private gIBSCBSMonoTot gMono;

    public double getvBCIBSCBS() {
        return vBCIBSCBS;
    }

    public void setvBCIBSCBS(double vBCIBSCBS) {
        this.vBCIBSCBS = vBCIBSCBS;
    }

    public gIBSTot getgIBS() {
        return gIBS;
    }

    public void setgIBS(gIBSTot gIBS) {
        this.gIBS = gIBS;
    }

    public gCBSTot getgCBS() {
        return gCBS;
    }

    public void setgCBS(gCBSTot gCBS) {
        this.gCBS = gCBS;
    }

    public gIBSCBSMonoTot getgMono() {
        return gMono;
    }

    public void setgMono(gIBSCBSMonoTot gMono) {
        this.gMono = gMono;
    }
}
